portal/static/portal/sass/partials/_banners.scss

```diff
@@ -131,8 +131,15 @@
   background-position: top left;
 }
 
+.banner--homepage--aimmo-materials,
+.banner--aimmo-materials {
+  background-image: url('../img/background_grey.png');
+  background-position: top right;
+}
+
 .banner--teacher,
-.banner--student {
+.banner--student,
+.banner--aimmo-materials {
```

Comment:
FYI There is a change in the level of granularity here (from teacher/student to more specific names). Is there a way to make this more generic so we can reuse the class names and have less to maintain in the future?

Reply:
I changed the name to describe the pattern - i hope it's better - aka: what would @ruthdw say? ;)